### DISCUSSION TITLE
Downtime for https://search.inetol.net

### DISCUSSION THREAD

**Discussion by inetol** [searxng/searx-instances#345] [Report instance downtime]
It is not a "downtime issue", but [this](https://status.inetol.net/cli1j19yr522061ban3bhs5vwo6) could happen again.

Reported issue:
> OSAI has reported multiple attacks occurred, starting on May 23rd 2023, 23:14:46 blocking all connections temporary to "search.inetol.net" due to a spike in requests.
>
> It is confirmed that the attack is being neutralized automatically and should not be affecting search providers to any great extent. An issue has been opened to improve the instance behavior and avoid possible outages in the future.

**Comment by unixfox** [comment]
Thank you for the notice.

I'm interested, what is OSAI?

**Reply by inetol** [reply]
Hey there, OSAI monitors and manages certain infrastructure software without the need for so much manual intervention, thus if I'm sleeping or traveling and something happens to the servers, they _should_ handle it on their own.

**Reply by unixfox** [reply]
it's an open source software?

**Reply by inetol** [reply]
Nope, for security reasons (and because it is totally integrated in the way I do things) I keep many things under lock and key in case there are undiscovered vulnerabilities. However, I don't discard the possibility of releasing the code someday.

**Comment by inetol** [comment]
(following the thread...)

There has been a [_small_](https://status.inetol.net/cliw8pphe47810amn1o9e524io) problem with the maintenance of DNS records and now in many countries my domain is not working at all. This is out of my hands and it's not like I can refresh the TTL of all resolvers manually.

I estimate that in 3 hours it _should_ be fixed in most countries, 6 hours and _should_ be fully working again.
